types/gulp-rename/index.d.ts

```diff
@@ -20,5 +21,6 @@ declare namespace rename {
 
 declare function rename(name: string): NodeJS.ReadWriteStream;
 declare function rename(callback: (path: rename.ParsedPath) => any): NodeJS.ReadWriteStream;
+declare function rename(callback: (path: rename.ParsedPath, file: File) => any): NodeJS.ReadWriteStream;
```

Comment:
I think this should actually be added to the previous overload. It is valid for a caller to pass a callback with less arguments, so documenting the second argument won't be a BC break.

Reply:
done